src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -20,6 +20,8 @@ final class HomeInteractor: RequiresAppDependencies {
 		self.homeViewController = homeViewController
 		self.state = state
 		self.exposureSubmissionService = exposureSubmissionService
+
+		updateAndReloadRiskCellState(to: riskProvider.activityState)
 		observeRisk()
```

Comment:
I wonder if observing the risk after starting updateAndReloading might lead to a race condition?
Isn't it better to first observerRisk() and after that trigger the update?

Reply:
I changed it to setting just the state instead of reloading the cell.
The cell gets loaded anyways after init.